src/components/TriviaListDisplayContainer/TriviaCardList.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { TriviaCard } from "./TriviaCard";
+import { useState } from "react";
+
+
+interface TriviaEntry {
```

Comment:
I suggest you separate this into a types file: TriviaCard.types.tsx. This way it will be easily reusable by other components/files that need to use this type without needing to import the entire TriviaCard component. 

This will save on computing because it minimizes imports AND it can help prevent circular import patterns.

Reply:
Example file path: 
/components/TriviaCard/index.tsx
/components/TriviaCard/TriviaCard.types.ts